lldb/test/API/functionalities/scripted_process/dummy_scripted_process.py

```diff
@@ -100,6 +111,10 @@ def get_register_context(self) -> str:
 
 def __lldb_init_module(debugger, dict):
     if not "SKIP_SCRIPTED_PROCESS_LAUNCH" in os.environ:
+        debugger.HandleCommand(
```

Comment:
Why do you have to add the stop hook here when the module is initialized and also in the test case?

Reply:
I added this for when loading the file it by hand when debugging, so I don't have to type the command every time I load it in an interactive debug session. As you can see it only takes this path when `SKIP_SCRIPTED_PROCESS_LAUNCH` is set, which is not the case in the test.